core/trino-main/src/main/java/io/trino/operator/BigintGroupByHash.java

```diff
@@ -60,8 +64,8 @@
     private int mask;
 
     // the hash table from values to groupIds
-    private LongBigArray values;
-    private IntBigArray groupIds;
+    private long[] values;
```

Comment:
nit: this could be an issue because `MultiChannelGroupByHash` will be used in synthetic context, while single channel aggregations are added by optimizer. However, there are ways to get around this if it becomes an issue